test/runtests.jl

```diff
@@ -112,6 +112,22 @@ const DATADIR = joinpath(@__DIR__, "data")
     @test pyconvert(Float64, ann["duration"]) == 0.256
     @test pyconvert(String, ann["description"]) == "type A"
 
+    # Modify it to have a start time with high precision
+    edf_high = @set edf.signals[end].records[1][2].onset_in_seconds = 1e-7
+    @test edf_high.signals[end].records[1][2].onset_in_seconds == 1e-7
+    py = mne_read(edf_high)
```

Comment:
this errors on main (or rather, on #71) because we hit the `failed to fit number into EDF's 8 ASCII character limit` path in `_edf_repr` here when going through `EDF.write`. On this branch it does not error because we just write it out with all digits, even if MNE will round them down later.